File: generate-pdf.js
const puppeteer = require('puppeteer');
const path = require('path');

async function generatePDF() {
    const browser = await puppeteer.launch();
    const page = await browser.newPage();
    
    // Load the HTML file
    const htmlPath = path.join(__dirname, 'index.html');
    await page.goto(`file://${htmlPath}`, { waitUntil: 'networkidle0' });
    
    // Generate PDF with optimized settings
    await page.pdf({
        path: 'Digivation_Company_Profile_2025.pdf',
        format: 'A4',
        printBackground: true,
        margin: {
            top: '0.5in',
            right: '0.5in',
            bottom: '0.5in',
            left: '0.5in'
        }
    });
    
    await browser.close();
    console.log('PDF generated successfully: Digivation_Company_Profile_2025.pdf');
}

generatePDF().catch(console.error);

File: generate_pdf.py
#!/usr/bin/env python3
import weasyprint
import os
from weasyprint import HTML, CSS

def generate_pdf():
    # Get the current directory
    current_dir = os.path.dirname(os.path.abspath(__file__))
    html_file = os.path.join(current_dir, 'index.html')
    pdf_file = os.path.join(current_dir, 'Digivation_Company_Profile_Landscape.pdf')
    
    # CSS for landscape orientation
    landscape_css = CSS(string='''
        @page {
            size: A4 landscape;
            margin: 0.5in;
        }
    ''')
    
    # Generate PDF
    html = HTML(filename=html_file)
    html.write_pdf(pdf_file, stylesheets=[landscape_css])
    
    print(f"Landscape PDF generated: {pdf_file}")

if __name__ == "__main__":
    generate_pdf()